src/guc.c

```diff
@@ -67,8 +67,11 @@ _guc_init(void)
 							"Maximum open chunks per insert",
 							"Maximum number of open chunk tables per insert",
 							&guc_max_open_chunks_per_insert,
-							work_mem * 1024L / 512L,	/* Assume each chunk
-														 * takes up 512 bytes
+							work_mem * 1024L / 25000L,	/* Measurements via
+														 * `MemoryContextStats(TopMemoryContext)`
+														 * show chunk insert
+														 * state memory context
+														 * takes up 24576 bytes
```

Comment:
Do we want the exact number here in the comments? This seems like something that can easily become silently out-of-date. At least say  ~25k.

Reply:
Fixed. (Honestly you're right, and 25K may even be too exact, but I want some number in there so the next person will know if it's time to change it)